tests/integration/test_backup_restore_on_cluster/configs/allow_database_types.xml

```diff
@@ -3,6 +3,9 @@
         <default>
             <allow_experimental_database_replicated>1</allow_experimental_database_replicated>
             <allow_deprecated_database_ordinary>1</allow_deprecated_database_ordinary>
+            <backup_keeper_max_retries>3</backup_keeper_max_retries>
+            <backup_keeper_retry_initial_backoff_ms>1</backup_keeper_retry_initial_backoff_ms>
+            <backup_keeper_retry_max_backoff_ms>1</backup_keeper_retry_max_backoff_ms>
```

Comment:
Please enable fault injections in this test as well. You can use a query setting `insert_keeper_fault_injection_probability` (it requires using `ZooKeeperWithFaultInjection` wrapper in code) or a zookeeper client config parameters `send_fault_probability` and `recv_fault_probability` (it will actually break session, but it's okay for integration tests). Also you can use `PartitionManager` with `probability` parameter in the test.

Reply:
I just lowered the default settings for retries to make `test_backup_restore_on_cluster/test.py::test_stop_other_host_during_backup` faster. Will add fault injection.

Reply:
The problem is that `ZooKeeperWithFaultInjection` doesn't implement many methods from `ZooKeeper` and if I enable `send_fault_probability` and `recv_fault_probability` globally then `ON CLUSTER` queries will fail. I don't want to change tests a log and replace `.query` to `.query_with_retry`

Reply:
> The problem is that ZooKeeperWithFaultInjection doesn't implement many methods from ZooKeeper

It should be easy to implement them

> then ON CLUSTER queries will fail

There are no `ON CLUSTER` queries in `test_backup_restore_new` and `test_backup_restore_s3`